setup.cfg

```diff
@@ -0,0 +1,22 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+
+[bumpversion:file:jsonschema_widget/__init__.py]
+
+[wheel]
+universal = 1
+
+[flake8]
+ignore = D203
+exclude =
+    jsonschema_widget/migrations,
+    .git,
+    .tox,
+    docs/conf.py,
+    build,
+    dist
+max-line-length = 119
```

Comment:
can we also add `isort` as in https://github.com/openwisp/django-freeradius/blob/master/setup.cfg#L17-L25 please?